EN.yaml

```
@@ -523,7 +523,7 @@ PSP_FIONA: Fiona
 PSP_FOXY: Foxy
 PSP_FREYA: Freya the Fateful
 PSP_GUARDIAN_LUCIFER: Guardian Lucifer
-PSP_HARLEQUIN: Harlequin
+PSP_HARLEQUIN: Harley Quinn

```

Harley Quinn is a knockoff of harlequin, not the other way around
```suggestion
PSP_HARLEQUIN: Harlequin

```